packages/phoenix-evals/src/phoenix/evals/utils.py

```diff
@@ -20,6 +25,16 @@
 _FUNCTION_NAME = "record_response"
 
 
+class AudioFormat(Enum):
+    WAV = "wav"
+
+
+@dataclass
+class Audio:
```

Comment:
this feels pretty hard for a user to really know they have to wrap the output in this wrapper class